ci-operator/step-registry/ibmcloud/deprovision/vpc/ibmcloud-deprovision-vpc-commands.sh

```diff
@@ -105,8 +105,16 @@ echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpc_name
 
 "${IBMCLOUD_CLI}" target -g ${resource_group} -r ${region}
 
-echo "DEBUG" "Removing the vpc ${vpc_name}"
+echo "DEBUG" "Removing the vpc ${vpc_name} ..."
 delete_vpc "${vpc_name}"
 
+echo "DEBUG" "Removing the resource reclamations ..."
+if [[ $("${IBMCLOUD_CLI}" resource reclamations -q) == "No reclamation found" ]]; then
+  echo "No reclamation found"
+else
+  ${IBMCLOUD_CLI} resource reclamations -q |  awk '(NR>1) {print $1}' | xargs -n1 ibmcloud resource reclamation-delete -f
```

Comment:
When already added `-q` option, still need `(NR>1)` in awk ?

Reply:
Can you show me an example output when reclamations are existing?

Reply:
[reclamation-delete succeed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36512/rehearse-36512-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-ibmcloud/1627675502225395712/artifacts/installer-rehearse-ibmcloud/ibmcloud-deprovision-vpc/build-log.txt)
the first row is "ID" ( ${IBMCLOUD_CLI} resource reclamations|awk '(NR>3) {print $1}') so got is 
> Submitting request to delete resource reclamation ID under account fdc2e14cf8bc4d53a67f972dc2e2c861 as ServiceId-dff277a9-b608-410a-ad24-c544e59e3778...
FAILED